src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncLocalsTests.cs

```diff
@@ -480,6 +480,13 @@ public static void Main()
 42
 ";
             CompileAndVerify(source, expectedOutput: expected, references: new[] { CSharpRef });
+
+            var comp = CodeGenAsyncTests.CreateRuntimeAsyncCompilation(source);
+            comp.VerifyEmitDiagnostics(
```

Comment:
Not directly related to this PR: how come we're not getting a warning for experimental feature, now that we're referencing real BCL?

Reply:
We don't check for this currently. There's a prototype comment.